internal/config/kubernetes/base/package-registry/deployment.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
       - name: package-registry
-        image: docker.elastic.co/package-registry/distribution:staging
+        image: docker.elastic.co/package-registry/distribution:production
```

Comment:
Maybe we should use the lite image here, or a vanilla one with a proxy to production. But this can be left for another PR.